backslash/contrib/slash_plugin.py

```diff
@@ -170,6 +170,32 @@ def test_start(self):
     def test_skip(self, reason=None):
         self.current_test.mark_skipped(reason=reason)
 
+    @slash.plugins.registers_on(None)
+    def is_session_exist(self, session_id):
+        try:
+            self.client.api.get('/rest/sessions/{0}'.format(session_id))
+            return True
+        except HTTPError:
```

Comment:
Not good enough. 404 errors should be handled differently than internal server errors, bad gateways etc.